packages/react/src/components/passwordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+
+// import core base styles
+import 'hds-core';
+import styles from './PasswordInput.module.scss';
+import { TextInput, TextInputProps } from '../textInput';
+import { IconEye, IconEyeCrossed } from '../../icons';
+
+export type PasswordInputProps = Omit<
+  TextInputProps,
+  'buttonIcon' | 'buttonAriaLabel' | 'onButtonClick' | 'children' | 'label' | 'type'
+> & {
+  /**
+   * The label for the input
+   */
+  label: string;
+  /**
+   * Boolean value for whether the component will include show / hide password button. Defaults to true.
+   */
+  includeShowPasswordButton?: boolean;
+  /**
+   * Boolean value for whether the component will initially reveal the password or not.
+   * Only applied when includeShowPasswordButton is true. Defaults to false.
+   */
+  initiallyRevealed?: boolean;
+  /**
+   * Aria label for reveal password button
+   */
+  revealPasswordButtonAriaLabel?: string;
+  /**
+   * Aria label for conceal password button
+   */
+  concealPasswordButtonAriaLabel?: string;
+  /**
+   * Type of the password input. Only applied when includeShowPasswordButton is false.
+   * Use this if you wish to not have show password button, or you wish to apply external show password button.
+   */
+  type?: string;
+};
+
+type textInputProps = {
```

Comment:
Types should start with Capital letter?

Reply:
- [ ] Change type name to start with a Capital letter